### PR TITLE
Feature/anand89/main thermistor mux cleanup

### PR DESCRIPTION
**Pull request by AdityaAsGithub**
Cooling code - been working in a single branch so now I'm making things organized again

### CODE REVIEW

File: common/faults/fault_config.json [hunk 1]
```diff
@@ -64,40 +64,94 @@
...
-                    "time_to_latch": 10,
-                    "time_to_unlatch": 10,
-                    "lcd_message": "battery pump flow brokey"
+                    "fault_name": "dt_r_temp_high",
+                    "max": 80,
+                    "min":-100,
+                    "priority": "warning",
+                    "time_to_latch": 1000,
+                    "time_to_unlatch": 1000,
+                    "lcd_message": "Right Drivetrain High (>80C)"
```
Comment (LukeOxley) [inline common/faults/fault_config.json:100]: Are these off of the DT gearbox thermistor or the temp from the motor controller? Maybe specify in the message.

File: source/main_module/car/car.c [hunk 2]
```diff
@@ -70,21 +73,21 @@ void carPeriodic()
...
     // Set Outputs that are purely combinational (do not depend on previous events)
     // Default values will typically reflect those in the IDLE state
-    car.torque_r.torque_left  = 0.0f;
-    car.torque_r.torque_right = 0.0f;
-    car.buzzer    = false;
-    car.sdc_close = true;
+    // car.torque_r.torque_left  = 0.0f;
+    // car.torque_r.torque_right = 0.0f;
+    // car.buzzer    = false;
+    // car.sdc_close = true;
```
Comment (LukeOxley) [inline source/main_module/car/car.c:79]: We can’t forget to uncomment the initialization code

File: source/main_module/car/car.c [hunk 3]
```diff
@@ -481,9 +485,11 @@ void monitorSDCPeriodic()
     static uint8_t index = 0;
     bool *nodes = (bool *) &sdc_mux;
 
-    *(nodes+index++) = PHAL_readGPIO(SDC_MUX_DATA_GPIO_Port, SDC_MUX_DATA_Pin);;
+    uint8_t stat =  (uint8_t) PHAL_readGPIO(SDC_MUX_DATA_GPIO_Port, SDC_MUX_DATA_Pin);
+
+    *(nodes+index++) = stat;
 
-    index = (index == 4) ? (index + 1) : index;
+    // index = (index == 4) ? (index + 1) : index;
```
Comment (LukeOxley) [inline source/main_module/car/car.c:492]: remove code if you’re done with it

File: source/main_module/main.c [hunk 4]
```diff
@@ -235,24 +242,24 @@ int main(void){
...
 
     /* Task Creation */
     schedInit(APB1ClockRateHz);
     configureAnim(preflightAnimation, preflightChecks, 60, 750);
 
-    // taskCreate(coolingPeriodic, 200);
+    taskCreate(coolingPeriodic, 100);
     taskCreate(heartBeatLED, 500);
-    taskCreate(monitorSDCPeriodic, 15);
+    taskCreate(monitorSDCPeriodic, 20);
```
Comment (LukeOxley) [inline source/main_module/main.c:254]: maybe have the scan occur quickly, but reduce the rate the message is sent (sent per each full scan) maybe every 100-500ms or so

File: source/main_module/main.c [hunk 5]
```diff
@@ -235,24 +242,24 @@ int main(void){
...
     // taskCreate(carHeartbeat, 100);
-    // taskCreate(carPeriodic, 15);
+    taskCreate(carPeriodic, 15);
     // taskCreate(wheelSpeedsPeriodic, 15);
     // taskCreate(updatePowerMonitor, 100);
-    // taskCreate(heartBeatTask, 100);
+    taskCreate(heartBeatTask, 100);
     // taskCreate(parseMCDataPeriodic, MC_LOOP_DT);
     // taskCreate(daqPeriodic, DAQ_UPDATE_PERIOD);
```
Comment (LukeOxley) [inline source/main_module/main.c:261]: daq plz :)

File: source/main_module/main.c [hunk 6]
```diff
@@ -365,6 +372,12 @@ void preflightAnimation(void) {
     }
 }
 
+void brak_buzz_test()
+{
+    PHAL_toggleGPIO(BRK_LIGHT_GPIO_Port, BRK_LIGHT_Pin);
+    PHAL_toggleGPIO(BUZZER_GPIO_Port, BUZZER_Pin);
+}
+
```
Comment (LukeOxley) [inline source/main_module/main.c:380]: cute

**Review by LukeOxley — CHANGES_REQUESTED** [review]
Couple teeny things and suggestions

File: source/main_module/car/car.c (hunk 2 above)
Comment (AdityaAsGithub) [reply source/main_module/car/car.c:79]: Yeah lol that is getting updated when I finish car.c

**Review by LukeOxley — APPROVED** [review]
lgtm

**Pull request merged**